src/Symfony/Component/Messenger/Bridge/Amqp/Transport/AmqpTransportFactory.php

```diff
@@ -29,7 +29,7 @@ public function createTransport(string $dsn, array $options, SerializerInterface
 
     public function supports(string $dsn, array $options): bool
     {
-        return 0 === strpos($dsn, 'amqp://');
+        return 0 === strpos($dsn, 'amqp://') || 0 === strpos($dsn, 'amqps://');
```

Comment:
In your original issue-  #37002 - you had been using `amqp+ssl`. Is that also valid?

Reply:
Definitly not, it's a mistake. Thanks for input, I've edited it 🤦